pkgs/by-name/go/got/package.nix

```diff
@@ -13,8 +13,16 @@
 , bison
 , autoPatchelfHook
 , testers
+, signify
+, withSsh ? false, openssh
+# Default editor to use when neither VISUAL or EDITOR are defined
+, withDefaultEditor ? false, nvi
+, defaultEditor ? null
 }:
 
+let
+  defaultEditor' = if (defaultEditor != null) then defaultEditor else nvi;
```

Comment:
* Is there a way to assert that `defaultEditor'` provides `meta.mainProgram`, so that `lib.getExe` used below will work properly?
* If `defaultEditor` is set then `withDefaultEditor` should be `true`, no?

Reply:
There's a better way to handle this, see latest changes to this PR